packages/filebrowser/src/listing.ts

```diff
@@ -941,6 +976,13 @@ export class DirListing extends Widget {
     const { width } =
       msg.width === -1 ? this.node.getBoundingClientRect() : msg;
     this.toggleClass('jp-DirListing-narrow', width < 250);
+
+    // Rerender item nodes' modified dates, if the modified style has changed.
+    const oldModifiedStyle = this._modifiedStyle;
+    this._updateModifiedSize(this.node);
```

Comment:
There is potential for layout trashing on 250px boundary here:
- we have read in line 977 (`this.node.getBoundingClientRect()`)
- write in line 978 (toggle class)
- read in `_updateModifiedSize()` via `getBoundingClientRect` on modified column
- write in `_updateModifiedSize()`

I am not sure if we can improve it much though, so no action requested.